cmd/numalign/main.go

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ */
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	flag "github.com/spf13/pflag"
+
+	"github.com/openshift-kni/debug-tools/internal/pkg/numalign"
+	"github.com/openshift-kni/debug-tools/internal/pkg/vfs"
+)
+
+func main() {
+	var sleepHoursParam = flag.StringP("sleep-hours", "S", "", "sleep hours once done.")
+	var scriptPathParam = flag.StringP("script-path", "P", "", "save test script to this path.")
+	var procfsRootParam = flag.StringP("procfs", "p", "/proc", "procfs root.")
+	var sysfsRootParam = flag.StringP("sysfs", "s", "/sys", "sysfs root.")
+	var debugParam = flag.BoolP("debug", "D", false, "enable debug mode.")
+	flag.Parse()
+
+	if _, ok := os.LookupEnv("NUMALIGN_DEBUG"); !ok || !*debugParam {
```

Comment:
IIUC we can set debug or via arguments or via ENV variable, so I think it should be &&

Reply:
probably yes, but this can be simplified a bit. I'll polish it.